queries.sql

```diff
@@ -7,6 +7,26 @@ SELECT name, date_of_birth FROM animals WHERE name = 'Agumon' OR name='Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg >= 10.5;
  SELECT name, neutered FROM animals WHERE neute
 red = TRUE;
-SELECT name FROM animaltests WHERE name <> 'Gabumon';
+SELECT name FROM animals WHERE name <> 'Gabumon';
 SELECT name, weight_kg FROM animals WHERE weig
 ht_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+ROLLBACK;
+
```

Comment:
- [ ] Kindly verify that the change was undone after ROLLBACK

- [ ] You're missing some parts of the query which are;
- [ ] Inside a transaction:

* `UPDATE` the animals table by setting the species column to digimon for all animals that have a name ending in mon.
* `UPDATE` the animals table by setting the species column to pokemon for all animals that don't have species already set.
* Commit the transaction. And don't forget to verify that change was made before commit and undone after commit.

- [ ] Inside a transaction delete all records in the animals table, then roll back the transaction. Please also verify that change was made before ROLLBACK and undone after ROLLBACK